lib/cli/utils.ts

```diff
@@ -33,6 +34,12 @@ export const coinsToSats = (coinsQuantity: number) => {
   return Math.round(coinsQuantity * SATOSHIS_PER_COIN);
 };
 
+/** Checks max decimals. */
```

Comment:
It's not clear what this does based on the description, it looks like earlier this returned `true` when the check passed, but now it returns `true` when the check fails. So better to specify here that the function returns true when the decimal places are exceeded.